lib/LANraragi/Plugin/Login/Fakku.pm

```diff
@@ -0,0 +1,60 @@
+package LANraragi::Plugin::Login::Fakku;
+
+use strict;
+use warnings;
+no warnings 'uninitialized';
+
+use Mojo::UserAgent;
+use LANraragi::Utils::Logging qw(get_logger);
+
+sub plugin_info {
+
+    return (
+        name      => "Fakku",
+        type      => "login",
+        namespace => "fakkulogin",
+        author    => "Nodja",
+        version   => "0.1",
+        description =>
+          "Handles login to fakku. The cookie is only valid for 7 days so don't forget to update it.",
```

Comment:
Having to update the cookie every 7 days kinda sucks, but I'm guessing the login procedure is too annoying to automate.   
Is `fakku_sid` the only cookie that's needed? I don't actually have a Fakku account so I haven't tested this.

Reply:
Yeah only fakku_sid is needed, the login form uses the a hidden google recaptcha field so I didn't even bother trying.